runEclipseHelper.sh

```diff
@@ -7,8 +7,9 @@ if [[ -f "$(pwd)/runEclipseHelper.sh" ]]; then
 	return 0
 fi
 
-if [[ -z $JAVA8 ]]; then
-	echo "JAVA8 environment variable not set. Please set it to the path of a Java 8 java executable"
+#Assuming the Java version is configured for JDK 21
+if [[ -z $JAVA_HOME ]]; then
+	echo "JAVA21 environment variable not set. Please set it to the path of a Java 21 java executable"
```

Comment:
Should this be `JAVA_HOME` to reflect what's actually tested?

```suggestion
	echo "JAVA_HOME environment variable not set. Please set it to the path of a Java 21 java executable"
```

Reply:
So the build config on Jenkins has been modified such that, upon triggering the build the variable `export JAVA_HOME = /opt/jdk21` holds precedence. This variable comes from our build configs and it points to the jdk21. 

I see your suggestion, going forward for next few releases we will be using jdk 21 for next couple of releases, unless new eclipse version used by KNIME is bumped to Java 22 or more. We can keep both, whatever you suggest?

Edit: I updated it